test/fixtures/users.yml

```diff
@@ -2,11 +2,11 @@
 user_00025: 
   salt: 2aa64c10d3ea67188845a59d3d9d411d52301838
```

Comment:
I believe this (and all similar lines in this file) needs to be changed to `password_salt` instead of `salt`.